From e7ed770363e8769727a915b2390c0ce1ab2ce964 Mon Sep 17 00:00:00 2001
Message-Id: <e7ed770363e8769727a915b2390c0ce1ab2ce964.1403136949.git.crobinso@redhat.com>
From: Michal Hlavinka <mhlavink@redhat.com>
Date: Wed, 18 Jun 2014 20:07:24 -0400
Subject: [PATCH] tests: Fix cookie comparison on rawhide

Cookie comments changed, so just strip them out

(crobinso: Add commit message, make change more readable)
---
 tests/misc.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/tests/misc.py b/tests/misc.py
index 4b4f99d..12da806 100644
--- a/tests/misc.py
+++ b/tests/misc.py
@@ -83,7 +83,12 @@ class MiscAPI(unittest.TestCase):
 
         # Mozilla should be converted inplace to LWP
         bugzilla.Bugzilla3(url=None, cookiefile=cookiesnew)
-        self.assertEquals(open(cookiesmoz).read(), open(cookiesnew).read())
+
+        def strip_comments(content):
+            return [l for l in content.split("\n") if not l.startswith("#")]
+        self.assertEquals(
+            strip_comments(open(cookiesmoz).read()),
+            strip_comments(open(cookiesnew).read()))
 
         # Make sure bad cookies raise an error
         try:
-- 
1.9.3

